modules/checking-cluster-resources-after-initiating-limited-live-migration.adoc

```diff
@@ -0,0 +1,179 @@
+// Module included in the following assemblies:
+//
+// * networking/ovn_kubernetes_network_provider/migrate-from-openshift-sdn.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="checking-cluster-resources-after-initiating-limited-live-migration_{context}"]
+= Checking cluster resources after initiating the limited live migration
+
+The following procedure shows you how to check metrics for EgressIPs resources, EgressFirewalls resources, and multicast enabled namespaces. If you have these resources, you should check them after migration to ensure their intended behavior.
+
+.Prerequisites
+
+* You have access to the cluster as a user with the `cluster-admin` role.
+* You have successfully migrated from OpenShift SDN to OVN-Kubernetes by using the limited live migration.
+
+.Procedure
+
+. As an {product-title} cluster administrator, check for EgressFirewall resources:
+
+.. Enter the following command to check for EgressFirewall resources:
++
+[source,terminal]
+----
+$ oc get egressfirewalls.k8s.ovn.org -A
+----
++
+.Example output
++
+[source,terminal]
+----
+NAMESPACE    NAME                      AGE
+<namespace>  <example_egress_policy>   5d
+----
+
+.. You can check the intended behavior of an EgressFirewall resource by passing in the `-o yaml` flag. For example:
++
+[source,terminal]
+----
+$ oc get egressfirewall <example_egress_policy> -n <namespace> -o yaml
+----
++
+.Example output
++
+[source,terminal]
+----
+apiVersion: k8s.ovn.org/v1
+kind: EgressFirewall
+metadata:
+  name: <example_egress_policy>
+  namespace: <namespace>
+spec:
+  egress:
+  - type: Allow
+    to:
+      cidrSelector: 192.168.0.0/16
+  - type: Deny
+    to:
+      cidrSelector: 0.0.0.0/0
+----
++
+Ensure that the behavior of this resource is intended because it could have changed after migration. For more information about EgressFirewalls, see "Configuring an egress firewall for a project".
+
+. Check your cluster for EgressIP resources:
+
+.. Enter the following command to list the namespace with EgressIP resources:
++
+[source,terminal]
+----
+$ oc get egressip -A
```

Comment:
@zhaozhanqi According to https://issues.redhat.com/browse/OCPBUGS-19542, this command can only list one egressIP. This could be misleading to customers if there are multiple IPs. Should we use `$ oc get egressip -o yaml` or `$ oc get egressip -o json` instead?

Reply:
we can use `oc get egressip -A` to list the namespaces which enabled egressip and then
`oc get egressip -o yaml -n $namespace` to check the details egressip information.

Reply:
For me, `oc get egress ip -A` does not list namespaces: 

[source,terminal]
```suggestion
$ oc get egressip -A
+
.Example output
+
[source,terminal]
----
NAME                                          EGRESSIPS     ASSIGNED NODE                                          ASSIGNED EGRESSIPS
egressip-46710-upgrade-ns   192.0.2.10   ip-10-0-42-79.us-east-2.compute.internal   10.0.55.193
----
```

Reply:
oh, sorry, yes, you are right, no need `-A`